index.ts

```diff
@@ -0,0 +1,29 @@
+
+import { serve } from 'https://deno.land/std@0.177.0/http/server.ts'
```

Comment:
This particular file has been removed in the latest PRs. It was added to remove the CORS error. But since it's working fine now, there isn't any need of this file, so removed it.